### PR TITLE
Add config file unit tests

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Adding tests before refactoring the logic.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@kubernetes-sigs/cri-tools-maintainers PTAL

File: pkg/common/file_test.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+`, &common.Config{
+		RuntimeEndpoint:   "foo",
+		ImageEndpoint:     "bar",
+		Timeout:           10,
+		Debug:             true,
+		PullImageOnCreate: true,
+		DisablePullOnRun:  true,
+	}, false),
+
+	Entry("should fail with invalid config option", `runtime-endpoint-wrong: "foo"`, nil, true),
```
Comment (SergeyKanzhelev) [inline pkg/common/file_test.go:69]: Would be great to have tests:

1. with the comments in the file
2. with empty lines between entries
3. with duplicated entries
4. empty file

All of them I believe are considered to be valid configs and a not that unusual to have

Comment (saschagrunert) [reply pkg/common/file_test.go:69]: Sure, adding them in https://github.com/kubernetes-sigs/cri-tools/pull/1648

**Pull request merged**